draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -308,11 +308,19 @@ and diagnosing the cause of the DNS filtering.
 
 ## Client Generating Request {#client-request}
 
-When generating a DNS query, the client includes the Extended DNS
-Error option Section 2 of {{!RFC8914}} in the OPT pseudo-RR {{!RFC6891}} to
-elicit the Extended DNS Error option in the DNS response. It SHOULD use an
-option-length of 0 (that is, omitting INFO-CODE and EXTRA-TEXT from
-{{!RFC8914}}).
+To prevent the EDE Option attacks to an EDE OPT-unaware DNS responder
+described in {{security}}, the client MUST use
+{{!I-D.ietf-add-resolver-info}} to determine whether the DNS responder supports
```

Comment:
I still think this is too strong. resinfo should be listed as an example here.

A client can have a policy to control the resolvers it communicates with.